src/components/cards/constants.ts

```diff
@@ -0,0 +1,184 @@
+      primaryTextColor: globalColors.white100,
+      secondaryTextColor: globalColors.blue20,
+    },
+    darkBlue: {
+      gradient: isDarkMode
+        ? [globalColors.blue100, globalColors.blue90]
+        : [globalColors.blue90, globalColors.blue80],
+      shadowColor: globalColors.blue90,
+      orbColorLight: globalColors.blue20,
+      orbColorDark: globalColors.blue100,
+      primaryTextColor: globalColors.white100,
+      secondaryTextColor: globalColors.blue20,
+    },
+    yellow: {
+      gradient: isDarkMode
+        ? [globalColors.yellow70, '#FFCB0F']
+        : ['#FFCB0F', '#FFDA24'],
+      shadowColor: '#FFCB0F',
+      orbColorLight: globalColors.yellow20,
+      orbColorDark: globalColors.yellow70,
+      primaryTextColor: globalColors.orange100,
+      secondaryTextColor: globalColors.yellow90,
+    },
+    green: {
+      gradient: isDarkMode
+        ? [globalColors.green70, globalColors.green60]
+        : [globalColors.green60, '#3ECF5B'],
+      shadowColor: globalColors.green60,
+      orbColorLight: globalColors.green20,
+      orbColorDark: globalColors.green70,
+      primaryTextColor: globalColors.white100,
+      secondaryTextColor: globalColors.green20,
+    },
+    darkGreen: {
+      gradient: isDarkMode
+        ? [globalColors.green100, globalColors.green90]
+        : [globalColors.green90, globalColors.green80],
+      shadowColor: globalColors.green90,
+      orbColorLight: globalColors.green20,
+      orbColorDark: globalColors.green100,
+      primaryTextColor: globalColors.white100,
+      secondaryTextColor: globalColors.green20,
+    },
+  };
+};
+
+export const learnCategoryColors: (
+  isDarkMode: boolean
+) => {
+  [key in LearnCategory]: CardColorway;
+} = (isDarkMode: boolean) => {
+  const themedCardColorways = cardColorways(isDarkMode);
+  return {
+    'Essentials': themedCardColorways.purple,
+    'Staying Safe': themedCardColorways.pink,
+    'How to use Rainbow': themedCardColorways.blue,
+    // 'How to use Rainbow 2': cardColorways.darkBlue,
+    "Beginner's Guides": themedCardColorways.yellow,
+    'What is Web3?': themedCardColorways.green,
+    'Blockchains and Fees': themedCardColorways.darkGreen,
+  };
+};
+
+const getStartedCard: LearnCardDetails = {
+  url: 'https://learn.rainbow.me/get-started-with-rainbow',
+  category: 'Essentials',
+  title: 'Get Started with Rainbow',
+  emoji: '🌈',
+  description:
+    "Welcome to Rainbow! We're so glad you're here. Weve created this guide to help with the basics of Rainbow and get you started on your new Web3 and Ethereum journey.",
+};
+
+const backupsCard: LearnCardDetails = {
+  url: 'https://learn.rainbow.me/the-importance-of-backups',
+  category: 'Essentials',
+  title: 'The Importance of Backups',
+  emoji: '☮️',
+  description:
+    'Keeping your wallet safe, secure, and backed up is essential to wallet ownership. Here we’ll chat about why it’s important to backup your wallet and the different methods that you can backup with.',
```

Comment:
yo no hablo ingles hermano

Reply:
sorry we only support english and french